### PR TITLE
[Added} The Features section was added

### PR DESCRIPTION
**Pull request by 0amirk**
The Features section was created using react and tailwind as mentioned in #3.

desktop-view
This is the desktop view.

tablet-view
Tablet view.

mobile-view
This is the mobile view.

Unnecessary files and folders were removed.

### CODE REVIEW

File: src/components/Features/index.jsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    return(
+        <div className="flex flex-col py-24 px-8">
+        {/* main div */}
+            <div className="text-center mb-16">
+            {/* text part */}
+                <p className="text-[#6941C6] text-base mb-3">
+                    Features
+                </p>
+
+                <p className=" text-4xl text-[#101828] mb-5">
```
Comment (kamalika0363) [inline src/components/Features/index.jsx:57]: font-semibold

File: src/components/Features/index.jsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+                Analytics that feels like it’s from the future
+
+                </p>
+
+                <p className="text-xl text-[#667085]">
```
Comment (kamalika0363) [inline src/components/Features/index.jsx:62]: sm:text-2xl sm:px-12 lg:px-72

**Comment by kamalika0363** [comment]
@0amirk make the changes

**Comment by 0amirk** [comment]
Made the changes.

**Comment by kamalika0363** [comment]
GG no more changes.
@0amirk shut down the VM.

**Comment by 0amirk** [comment]
_Thank god_

**Pull request merged**